samples/nrf9160/modem_shell/src/location/location_cmd_utils.c

```diff
@@ -71,7 +71,7 @@ static int location_cmd_utils_generate_nmea(const struct location_data *location
 #define MSG_APP_ID_GNSS_FORMAT_PVT  "GNSS"
 #define MSG_TYPE_DATA               "DATA"
 
-#define MSG_TYPE_DATA_GNSS_LONGITUDE "lng"
+#define MSG_TYPE_DATA_GNSS_LONGITUDE "lon"
```

Comment:
This is not actually related to your change, but I think it would be good to change the order in the implementation so, that latitude comes first. That's the way the coordinates are always given. In addition to the defines here, also the field order in the JSON should be changed.

Reply:
Good point. I'll change the order of those.